Add section calling out realtime streams (#982)
snowplow/documentation: docs/getting-started-on-community-edition/quick-start/index.md

  </TabItem>
</Tabs>

> ## Warehouse storage options

Reviewer: ```suggestion
## Storage options
```

| BigQuery | — | :white_check_mark: | — |
| Synapse Analytics | — | — | :white_check_mark: |

> ## Real-Time Streaming Storage Options

Reviewer 2: Should be "Real-time streaming storage options" (note the lower case letters) for consistency with other titles...

> ## Real-time streaming storage options

Reviewer: ```suggestion
## Real-time streaming options
```

As part of the deployment, your data will be available in real-time streams corresponding to the cloud provider you have chosen. You can consume data directly from these streams, either in addition to or instead of the data warehouse.

| Stream | AWS | GCP | Azure |
|:----------|:---:|:---:|:-----:|
| Kinesis | :white_check_mark: | :x: | :x: |
| Pub/Sub | :x: | :white_check_mark: |:x: |
| EventHubs | :x: | :x: | :white_check_mark: |

> For an out-of-the-box solution to accessing this data in real-time streams, you can [check out our Snowbridge project](/docs/destinations/forwarding-events/snowbridge/). Alternatively, if you want to develop a custom consumer, you can [leverage our Analytics SDKs](https://docs.snowplow.io/docs/destinations/analytics-sdk/) to parse the event formats more easily.

Reviewer: ```suggestion
For an out-of-the-box solution to accessing this data in real-time streams, you can [check out our Snowbridge project](/docs/destinations/forwarding-events/snowbridge/index.md). Alternatively, if you want to develop a custom consumer, you can [leverage our Analytics SDKs](/docs/destinations/analytics-sdk/index.md) to parse the event formats more easily.
```
